content/topics/work/mock-interview-feedback/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+ready: true
+title: Mock Interview Feedback
+---
+### Mock Interview  Feedback
```

Comment:
Headings are repetitive. Please look at the web page and see how this is rendered